tests/instance/assess_instance_runs.py

```diff
@@ -1,25 +1,26 @@
+import datetime
+
 import flywheel
 import pandas as pd
 
+versions = ["1.0.0_1.0.20200331", "1.1.0_1.0.20201102_dev0"]
```

Comment:
I am only placing these scripts with the repo to keep them together and not lose anything. As you know, instance tests are not yet automated because we do not yet have a solution for end-to-end testing of Gears. Therefore, these scripts are really just for my testing of the Gear, they are not meant to be "production-ready".